sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/implementation/models/CertificateCreateParameters.java

```diff
@@ -145,11 +146,45 @@ public static CertificateCreateParameters fromJson(JsonReader jsonReader) throws
                 } else if ("tags".equals(fieldName)) {
                     Map<String, String> tags = reader.readMap(reader1 -> reader1.getString());
                     deserializedCertificateCreateParameters.tags = tags;
+                } else if ("preserveCertOrder".equals(fieldName)) {
+                    deserializedCertificateCreateParameters.preserveCertOrder
+                        = reader.getNullable(JsonReader::getBoolean);
                 } else {
                     reader.skipChildren();
                 }
             }
             return deserializedCertificateCreateParameters;
         });
     }
+
+    /*
+     * Specifies whether the certificate chain preserves its original order. The default value is false, which sets the
+     * leaf certificate at index 0.
+     */
+    @Generated
+    private Boolean preserveCertOrder;
+
+    /**
+     * Get the preserveCertOrder property: Specifies whether the certificate chain preserves its original order. The
+     * default value is false, which sets the leaf certificate at index 0.
+     *
+     * @return the preserveCertOrder value.
+     */
+    @Generated
+    public Boolean isPreserveCertOrder() {
+        return this.preserveCertOrder;
+    }
+
+    /**
+     * Set the preserveCertOrder property: Specifies whether the certificate chain preserves its original order. The
+     * default value is false, which sets the leaf certificate at index 0.
+     *
+     * @param preserveCertOrder the preserveCertOrder value to set.
+     * @return the CertificateCreateParameters object itself.
+     */
+    @Generated
+    public CertificateCreateParameters setPreserveCertOrder(Boolean preserveCertOrder) {
```

Comment:
These are new properties added to the spec but KV is generating these models in implementation package. So, these properties  will have to be manually mapped to public models.

Reply:
I think I will need to revert the changes in KV, and let dev decide how to update.

I think they've added "7.6-preview.2", and code generated here is from 7.6-preview.2.
https://github.com/Azure/azure-rest-api-specs/commits/main/specification/keyvault/Security.KeyVault.Certificates

But the `ServiceVersion` class in these modules is handwritten, hence I didn't find this change when only looking at SDK code changes.